blazor-toc.html

```diff
@@ -1497,6 +1497,15 @@
 					<li>
 						<a href="/blazor/datagrid/how-to/css-isolation-for-grid">CSS Isolation for DataGrid</a>
 					</li>
+					<li>
+						<a href="/blazor/datagrid/how-to/hide-checkbox-in-column-header">Hide CheckBox in Column Header</a>
+					</li>
+					<li>
+						<a href="/blazor/datagrid/how-to/auto-update-calculative-columns-in-batch-editing">Auto-Update Calculative Columns in Batch Editing</a>
+					</li>
+					<li>
+						<a href="/blazor/datagrid/how-to/dynamic-object-binding-with-edit-template">DynamicObject binding with edit template</a>
```

Comment:
move this topic at the last of https://blazor.syncfusion.com/documentation/datagrid/cell-edit-types#custom-editors-using-templatecell-edit-template